spec/example_app/db/seeds.rb

```diff
@@ -12,28 +12,40 @@
 Product.destroy_all
 ProductMetaTag.destroy_all
 Series.destroy_all
+Country.destroy_all
 
-100.times do
+countries = Country.create! [
```

Comment:
@nickcharlton Sorry, i don't understand what do you mean.
Something wrong with brackets?
I can separate attributes declaration from `.create!` like this:
```ruby
country_attributes = [
  { code: "US", name: "USA" },
  { code: "CA", name: "Canada" },
  { code: "CN", name: "China" },
  { code: "RU", name: "Russia" },
  { code: "AU", name: "Australia" },
]

countries = Country.create!(country_attributes)
```